src/styles/font.css

```
@@ -1,13 +1,70 @@
 }
+
+/* html 이 font-sans 를 상속시키므로, 상속값을 덮으려면 컨테이너에 직접 건다.
+   var(--font-sans) 를 재정의하는 방식은 html 에서 이미 확정된 상속값을
+   되돌리지 못한다. 전역 교체가 아니라 /about 로 범위를 좁힌 이유는, Pretendard
+   가 앱 기본 폰트라 다른 화면의 동적 텍스트(이름·학교명·프로젝트 제목)까지
+   서브셋에 없는 글자로 새어 나가기 때문이다. */
+.about-landing {
+  font-family:
+    "Pretendard About",
+    "Pretendard Variable",
+    "Pretendard",
```

Stylelint (17.14.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)